### PR TITLE
Completed Dashboard View

### PR DESCRIPTION
**Pull request by snholmes**
Completed dashboard view minus the snapshot and nav bar. Used static data for now.

### CODE REVIEW

File: SuiteCareers/SuiteCareers/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - SuiteCareers</title>
+    <title>@ViewData["Title"] | SuiteCareers</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/SuiteCareers.styles.css" asp-append-version="true" />
 </head>
-<body>
+<body style="background-color: #fbfcfe">
```
Comment (Bansenauer-Cascadia) [inline SuiteCareers/SuiteCareers/Views/Shared/_Layout.cshtml]: Could you put this in the main style sheet

**Review by Bansenauer-Cascadia — CHANGES_REQUESTED** [review]

**Review by darthcod3r — DISMISSED** [review]
The dashboard looks amazing Shelby! You did awesome!

File: SuiteCareers/SuiteCareers/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (snholmes) [reply SuiteCareers/SuiteCareers/Views/Shared/_Layout.cshtml]: Sure. The styling has been moved to site.css

**Review by ParkerW98 — APPROVED** [review]
This looks great Shelby!

**Review by ChaidGPT — APPROVED** [review]
Looks good!

**Review by Bansenauer-Cascadia — APPROVED** [review]

**Pull request merged**